src/index.js

```diff
@@ -48,9 +49,18 @@ export default declare(({
     if (typeof importPath !== 'string') {
       throw new TypeError('`applyPlugin` `importPath` must be a string');
     }
-    const { ignorePattern, caseSensitive, filename: providedFilename } = state.opts;
+    const {
+      ignorePattern, includePattern, caseSensitive, filename: providedFilename,
```

Comment:
```suggestion
      ignorePattern,
      includePattern,
      caseSensitive,
      filename: providedFilename,
```
:-) please don't use prettier on a repo that doesn't have it configured